statement.go

```diff
@@ -112,7 +112,7 @@ var (
 
 	// Other
 	exitRe            = regexp.MustCompile(`(?is)^EXIT$`)
-	useRe             = regexp.MustCompile(`(?is)^USE\s+(.+)$`)
+	useRe             = regexp.MustCompile(`(?is)^USE\s+(.+?)(?:\s+ROLE\s+(.+))?$`)
```

Comment:
`USE\s+(.+?)` looks very tricky to me. Also it can match the invalid statement like `USE db ROLE`.

How about using `USE\s+([^\s]+)` instead?

Reply:
Thank you for catching. Fixed as you commented.